storage/maria/aria_chk.c

```diff
@@ -236,9 +236,6 @@ int main(int argc, char **argv)
   }
   maria_end();
   my_exit(error);
-#ifndef _lint
```

Comment:
Keep the return 0;
(This was an #ifndef _lint, which means the code should be there for normal builds)

Reply:
After a no-return `my_exit` function the `return 0` is just dead code. Compilers and static analysers rightly complain about after that, normal build or not.